### PR TITLE
qemu.tests: Add data-plane support for virtio-scsi since RHEL.7.3.

### PR DESCRIPTION
**Pull request by CongLi**
Signed-off-by: Cong Li <coli@redhat.com>

id: 1374122

### CODE REVIEW

**Comment by CongLi** [comment]
-device virtio-scsi-pci,iothread=iothread0,id=virtio_scsi_pci0,bus=pci.0,addr=03 \
    -drive id=drive_image1,if=none,snapshot=off,aio=native,cache=none,format=qcow2,file=/usr/share/avocado/data/avocado-vt/images/RHEL-Server-7.3-64-virtio-scsi.qcow2 \
    -device scsi-hd,id=image1,drive=drive_image1 \

File: qemu/tests/cfg/block_stream.cfg [hunk 1]
```diff
@@ -75,5 +75,10 @@
             variants:
                 - @default:
                 - data_plane:
+                    only Host_RHEL.m7
                     iothreads = iothread0
-                    blk_extra_params_image1 = "iothread=${iothreads}"
+                    virtio_blk:
+                        blk_extra_params_image1 = "iothread=${iothreads}"
+                    virtio_scsi:
+                        no Host_RHEL.m7.u0, Host_RHEL.m7.u1, Host_RHEL.m7.u2
```
Comment (suqinhuang) [inline qemu/tests/cfg/block_stream.cfg]: how about rhel6

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

File: qemu/tests/cfg/multi_host.cfg [hunk 2]
```diff
@@ -408,13 +408,16 @@
                     filesize = 100
                     transfer_timeout = 1000
 
-                - virtio_blk_data_plane:
+                - with_data_plane:
```
Comment (xutian) [inline qemu/tests/cfg/multi_host.cfg]: It looks below configuration can descriptor what you want to get.
```
only virtio-blk, virtio_scsi
bus_extra_params_image1 = "iothread=${iothreads}"
virtio_scsi:
       no Host_RHEL.m7.u0, Host_RHEL.m7.u1, Host_RHEL.m7.u2
```

Comment (CongLi) [inline qemu/tests/cfg/multi_host.cfg]: @xutian 
Fair enough.
And code has been updated, could you please review it again?

Thanks.

**Comment by xutian** [comment]
Case name changed, please ensure all test loops pickup right test case.

**Comment by CongLi** [comment]
Thanks @xutian .

Have checked, this PR changes won't influence any test loop.

**Comment by xutian** [comment]
looks good to me.

**Comment by CongLi** [comment]
Thanks @suqinhuang @xutian , merged.

**Pull request merged**